framework/aster-frame/src/bus/mmio/mod.rs

```diff
@@ -37,15 +37,18 @@ pub fn init() {
             tdx_guest::unprotect_gpa_range(0xFEB0_0000, 4).unwrap();
         }
     }
+
     // FIXME: The address 0xFEB0_0000 is obtained from an instance of microvm, and it may not work in other architecture.
+    #[cfg(target_arch = "x86_64")]
     iter_range(0xFEB0_0000..0xFEB0_4000);
```

Comment:
Iterate all the MMIO space may be unique to x86, maybe it needs a refactoring in the x86 side?

Reply:
To address this issue, there are several tasks that we need to complete, and I am currently working on them:

1. Implement the Interrupt Remapping function in IOMMU
2. #835 
3. Parsing device tree to get the MMIO information.
4. Removing the MMIO bus in the framework.

I think it's okay to add `#[cfg(target_arch="x86_64")]` as a workaround.

Reply:
> 3\. Parsing device tree to get the MMIO information.

As far as I know, device trees are typically NOT used on x86. Instead, device information are obtained from ACPI.